packages/command/src/sfdxFlags.ts

```diff
@@ -66,7 +77,9 @@ function option<T>(
   options: flags.Option<T>,
   parse: (val: string, ctx: unknown) => T
 ): flags.Discriminated<flags.Option<T>> {
-  return merge(kind, OclifFlags.build(Object.assign(options, { parse }))(), options);
+  const flag = OclifFlags.option({ ...options, parse });
```

Comment:
This is similar to how the `buildBoolean` uses `OclifFlags.boolean(options)`. I think this is also better than creating a builder with only the parse for initial options, and then using the options called immediately on the builder. This cuts out creating the flag builder and just creates the flag directly.

Reply:
Seems good.